src/onegov/org/forms/event.py

```diff
@@ -653,3 +653,20 @@ class EventConfigurationForm(Form):
             'class_': 'formcode-select',
             'data-fields-include': 'radio,checkbox'
         })
+
+    def validate(self):
+        """
+        Ensures none of the active keywords is used in the `EventForm` to
+        prevent overriding data accidentally by defining filters.
+        """
+        result = super().validate()
+
+        for keyword in [k.lower() for k
+                        in self.keyword_fields.data.splitlines()]:
+            if keyword in dir(EventForm):
```

Comment:
```suggestion
            if keyword in EventForm:
```
`Form` implements `__contains__` for field names, `dir` should be unnecessary. `dir` is a little bit expensive, so it's better if we can get away with not using it.